.github/workflows/test.yml

```diff
@@ -28,9 +28,13 @@ jobs:
           - go: '1.12'
             mongo: 'mongodb-linux-x86_64-3.6.20'
             minio: '2019-05-23T00-29-34Z'
+            wired_tiger: 'true'
+          - go: '1.12'
+            mongo: 'mongodb-linux-x86_64-ubuntu2204-7.0.4'
+            minio: '2019-05-23T00-29-34Z'
             wired_tiger: 'false'
           - go: '1.12'
-            mongo: 'mongodb-linux-x86_64-3.6.20'
+            mongo: 'mongodb-linux-x86_64-ubuntu2204-7.0.4'
             minio: '2019-05-23T00-29-34Z'
             wired_tiger: 'true'
```

Comment:
There's 4 test matrix entries now - I think we only need 2
* current production (mongo 3.6.12, wired tiger off)
* mongo 7, wired tiger is irrelevant since it's always on in 4+

Reply:
👍

Reply:
:+1: